src/Illuminate/Notifications/DatabaseNotification.php

```diff
@@ -52,7 +53,7 @@ public function notifiable()
      */
     public function markAsRead()
     {
-        $this->forceFill(['read' => true])->save();
+        $this->forceFill(['read' => Carbon::now()])->save();
```

Comment:
Just a thought, could do `$this->freshTimestamp()` here instead - would give you a new Carbon instance and save needing to add the use statement.